sites/fast-website/package.json

```diff
@@ -6,7 +6,10 @@
   "private": true,
   "license": "MIT",
   "devDependencies": {
-    "@microsoft/fast-components": "^0.10.1",
+    "@microsoft/fast-element": "0.9.0",
```

Comment:
I think we’ll ultimately want to opt into semver. If the publish again we may have the same issue but we’ll definitely want to let lerna handle this once it’s in master.

Reply:
Is there a reason we are moving to dev deps here?

Reply:
I believe these should just stay as dep

Reply:
@eljefe223 @chrisdholt Installing them as `dependencies` would leave them hanging out in `node_modules` for no real reason on production, since the packages are only needed for building. Once the `dist` directory is built, there's no need for any other packages in order to run the site.